### PR TITLE
add vp8 vp9

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

**Comment by retokromer** [comment]
LGTM

File: codec_specs.md [hunk 1]
```diff
@@ -220,6 +220,26 @@ Initialisation: The `Private Data` contains the fourcc as found in MP4 movies:
...
+
+Initialisation: none
+
+### V_VP9
+
+Codec ID: V_VP9
+
+Codec Name: VP9 Codec format
+
+Description: VP9 is an open and royalty free video compression format developed by Google as a successor to VP8. [Draft VP9 Bitstream Overview](https://www.ietf.org/archive/id/draft-grange-vp9-bitstream-00.txt)
```
Comment (robUx4) [inline codec_specs.md]: I would rather use the more generic link http://www.webmproject.org/vp9/
It has the bitstream and how to put it in different containers. Also the draft is more recent.

Comment (dericed) [inline codec_specs.md]: updated

**Comment by dericed** [comment]
ready for re-review

**Comment by retokromer** [comment]
LGTM

**Comment by robUx4** [comment]
Merged

**Pull request merged**